### ISSUE TITLE
Next steps?

### ISSUE THREAD

**Issue by Charminbear** [chaijs/type-detect#6]
@logicalparadox @keithamus 
I was wondering how are the next steps here. Are you going to release a new version? Or you want me to move `chai` to the current master branch? 

Also, I found one more little mistake in the docs:

`assert(type(new String('hello')) === 'object');` -> that should be 'string'
Its probably easier now to fix that directly on master than for me doing a pull request. ;)

**Comment by keithamus** [comment]
@Charminbear everything is done with PRs, so its best to send a PR for any failing documentation.

As for the next step - @logicalparadox will need to make a release, or set up the .travis.yml so releases can be made via PR, like Chai is.

Once that's done, we'll need a PR in Chai to remove its internal type detect and include type-detect in its package.json.

**Comment by jokeyrhyme** [comment]
I just noticed the `new String('blah')` issue, and that there's a commit for it in the master branch.

Would it be possible to `git tag` and `npm publish` a release that includes this fix? Cheers! <3

**Comment by keithamus** [comment]
@jokeyrhyme we will as soon as we can. Thanks

**Comment by logicalparadox** [comment]
@keithamus I have added in the necessary travis data to allow for tagged releases. I'll let you release to test.

**Comment by Charminbear** [comment]
Did you `npm publish`? On NPM, it still shows Version 0.1.2. (https://www.npmjs.com/package/type-detect)

**Comment by keithamus** [comment]
Sorry @Charminbear. @logicalparadox and I are resolving some build issues. It'll be done ASAP.

**Comment by Charminbear** [comment]
Alright, no problem. Good luck with this!

**Comment by jokeyrhyme** [comment]
You can put the following in your package.json file until the v1.0.0 is published to NPM:

```json
{
  "dependencies": {
    "type-detect": "chaijs/type-detect#1.0.0"
  }
}
```

**Comment by keithamus** [comment]
Good point @jokeyrhyme. We hope to get this released soon at which point we'll update chai :smile:

**Comment by logicalparadox** [comment]
I published `1.0.0` manually, but also updated the travis key so auto-publishing should work now.

/cc @keithamus

**Comment by keithamus** [comment]
:star2:
